File: 3172-divisible-and-non-divisible-sums-difference/3172-divisible-and-non-divisible-sums-difference.py
class Solution:
    def differenceOfSums(self, n: int, m: int) -> int:
        sum1 = 0
        sum2 = 0
        for x in range(1, n+1):
            if x%m == 0:
                sum1+=x
            else:
                sum2+=x
        return sum2-sum1

File: 3172-divisible-and-non-divisible-sums-difference/3172-divisible-and-non-divisible-sums-difference.kt
class Solution {
    fun differenceOfSums(n: Int, m: Int): Int {

        var sum1 = 0
        var sum2 = 0
        for (x in 1..n) {
            if (x%m == 0) sum1+=x
            else sum2+=x
        }
        return sum2-sum1
    }
}